<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QStandardItemModelDemoClass</class>
 <widget class="QMainWindow" name="QStandardItemModelDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QStandardItemModelDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>TableView</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableView" name="tvShow"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>PlainTextEdit</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="ptShow"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenFile"/>
   <addaction name="actSaveAs"/>
   <addaction name="actPreview"/>
   <addaction name="separator"/>
   <addaction name="actAddRow"/>
   <addaction name="actInsertRow"/>
   <addaction name="actDelRow"/>
   <addaction name="separator"/>
   <addaction name="actAlignLeft"/>
   <addaction name="actAlignCenter"/>
   <addaction name="actAlignRight"/>
   <addaction name="actBlod"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpenFile">
   <property name="icon">
    <iconset resource="QStandardItemModelDemo.qrc">
     <normaloff>:/QStandardItemModelDemo/Resources/MYSL.ICO</normaloff>:/QStandardItemModelDemo/Resources/MYSL.ICO</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="icon">
    <iconset resource="QStandardItemModelDemo.qrc">
     <normaloff>:/QStandardItemModelDemo/Resources/RESENDL.ICO</normaloff>:/QStandardItemModelDemo/Resources/RESENDL.ICO</iconset>
   </property>
   <property name="text">
    <string>另存文件</string>
   </property>
   <property name="toolTip">
    <string>另存文件</string>
   </property>
  </action>
  <action name="actPreview">
   <property name="icon">
    <iconset resource="QStandardItemModelDemo.qrc">
     <normaloff>:/QStandardItemModelDemo/Resources/REFBAR.ICO</normaloff>:/QStandardItemModelDemo/Resources/REFBAR.ICO</iconset>
   </property>
   <property name="text">
    <string>模型数据预览</string>
   </property>
   <property name="toolTip">
    <string>模型数据预览</string>
   </property>
  </action>
  <action name="actAddRow">
   <property name="icon">
    <iconset resource="QStandardItemModelDemo.qrc">
     <normaloff>:/QStandardItemModelDemo/Resources/DOCL.ICO</normaloff>:/QStandardItemModelDemo/Resources/DOCL.ICO</iconset>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
   <property name="toolTip">
    <string>添加行</string>
   </property>
  </action>
  <action name="actInsertRow">
   <property name="icon">
    <iconset resource="QStandardItemModelDemo.qrc">
     <normaloff>:/QStandardItemModelDemo/Resources/POSTITL.ICO</normaloff>:/QStandardItemModelDemo/Resources/POSTITL.ICO</iconset>
   </property>
   <property name="text">
    <string>插入行</string>
   </property>
   <property name="toolTip">
    <string>插入行</string>
   </property>
  </action>
  <action name="actDelRow">
   <property name="icon">
    <iconset resource="QStandardItemModelDemo.qrc">
     <normaloff>:/QStandardItemModelDemo/Resources/TASKDECL.ICO</normaloff>:/QStandardItemModelDemo/Resources/TASKDECL.ICO</iconset>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
   <property name="toolTip">
    <string>删除行</string>
   </property>
  </action>
  <action name="actAlignLeft">
   <property name="icon">
    <iconset resource="QStandardItemModelDemo.qrc">
     <normaloff>:/QStandardItemModelDemo/Resources/RSSITEML.ICO</normaloff>:/QStandardItemModelDemo/Resources/RSSITEML.ICO</iconset>
   </property>
   <property name="text">
    <string>居左</string>
   </property>
   <property name="toolTip">
    <string>居左</string>
   </property>
  </action>
  <action name="actAlignCenter">
   <property name="icon">
    <iconset resource="QStandardItemModelDemo.qrc">
     <normaloff>:/QStandardItemModelDemo/Resources/CNFNOT.ICO</normaloff>:/QStandardItemModelDemo/Resources/CNFNOT.ICO</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
   <property name="toolTip">
    <string>居中</string>
   </property>
  </action>
  <action name="actAlignRight">
   <property name="icon">
    <iconset resource="QStandardItemModelDemo.qrc">
     <normaloff>:/QStandardItemModelDemo/Resources/EXITEML.ICO</normaloff>:/QStandardItemModelDemo/Resources/EXITEML.ICO</iconset>
   </property>
   <property name="text">
    <string>居右</string>
   </property>
   <property name="toolTip">
    <string>居右</string>
   </property>
  </action>
  <action name="actBlod">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QStandardItemModelDemo.qrc">
     <normaloff>:/QStandardItemModelDemo/Resources/SECURL.ICO</normaloff>:/QStandardItemModelDemo/Resources/SECURL.ICO</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="QStandardItemModelDemo.qrc">
     <normaloff>:/QStandardItemModelDemo/Resources/OOFL.ICO</normaloff>:/QStandardItemModelDemo/Resources/OOFL.ICO</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QStandardItemModelDemo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>QStandardItemModelDemoClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
